application/ui/components/param-row.jsx

```diff
@@ -0,0 +1,288 @@
+                inlineParam = (
+                    <Select
+                        value    = {this.props.value}
+                        key      = {this.props.key}
+                        options  = {enumValues}
+                        onChange = {this.props.onChange}
+                    />
+                );
+                break;
+            case 'boolean':
+                inlineParam = (
+                    <Select
+                        value    = {this.props.value}
+                        key      = {this.props.key}
+                        options  = {[{value: false, label: 'false'}, {value: true, label: 'true'}]}
+                        onChange = {this.props.onChange}
+                    />
+                );
+                break;
+            case 'string':
+            case 'number':
+            case 'integer':
+            case 'float':
+                inlineParam = (
+                    <Text
+                        key      = {this.props.key}
+                        type     = {this.props.param.type}
+                        onChange = {this.props.onChange}
+                    />
+                );
+                break;
+            case 'checkbox':
+                inlineParam = (
+                    <Checkbox
+                        value    = {this.props.value}
+                        key      = {this.props.key}
+                        onChange = {this.props.onChange}
+                    />
+                );
+                break;
+            case 'stripe-token':
+                inlineParam = (
+                    <StripeToken
+                        paramName = {this.props.param.name}
+                        onChange  = {this.props.onChange}
+                    />
+                );
+                break;
+            case 'file':
+                if (this.props.method === 'GET') {
+                    rowParam = (<tr><td>{'INVALID CONFIGURATION - CANNOT USE `file` IN GET REQUEST'}</td></tr>);
+                } else {
+                    inlineParam = (
+                        <FileParam
+                            name = {this.props.param.name}
+                            onChange = {this.props.onChange}
+                        />
+                    );
+                }
+                break;
+        }
+
+        if (this.props.simple === true) {
+            return (
+                <tbody>
+                    <tr>
+                        <td><code>{this.props.param.name}</code></td>
+                        <td><code>{this.props.param.type}</code></td>
+                        <td>{inlineParam}</td>
+                    </tr>
+                    {rowParam}
+                </tbody>
+            );
+        }
+        description = this.renderDescription(this.props.param);
+
+        if (description) {
+            description = (
+                <tr>{description}</tr>
+            );
+        }
+
+        return (
+            <tbody className = {"paramrow"}>
```

Comment:
Small detail, but when the tag is all on one line the pattern we've settled on is:
```JSX
<tbody className='paramrow'>
```